tests/qam-minimal/install_update.pm

```diff
@@ -32,7 +30,7 @@ sub run {
 
     zypper_call("ref");
 
-    zypper_call(qq{in -l -y -t patch \$(zypper patches | awk -F "|" '/test-minimal/ { print \$2;}')}, exitcode => [0, 102, 103], log => 'zypper.log');
+    zypper_call('in -l -t patch ' . get_var('INCIDENT_PATCH'), exitcode => [0, 102, 103], log => 'zypper.log');
```

Comment:
why new function around  around zypper_call? this variant is flexible and has simple usage..

+ http://paste.opensuse.org/75168613 -- how is zypper in reality used in our tests ..